Lombiq.HelpfulLibraries.LinqToDb/Lombiq.HelpfulLibraries.LinqToDb.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
+		<DefaultItemExcludes>$(DefaultItemExcludes);.git*</DefaultItemExcludes>
+		<RootNamespace>Lombiq.HelpfulLibraries.Linq2Db</RootNamespace>
+	</PropertyGroup>
+	
+	<ItemGroup>
+		<FrameworkReference Include="Microsoft.AspNetCore.App" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="linq2db" Version="3.3.0" />
+		<PackageReference Include="OrchardCore.Data.Abstractions" Version="1.0.0-rc2-13929" />

```

Comment:
Még az OrchardCore.Data.YesSql is kell. Nem tudom, miért nem kapunk itt build errort, más projectben próbálva lesz.

Reply:
Milyen verziószámú kellene? Mert a mostanival tele lettem errorral.

Reply:
Hmm, akkor lehet, hogy valami másképp van ebben a verzióban, mert most a 16154-ben most hiba volt. De amúgy a Finitive-ben nem tudtam reprózni, szal akkor valszeg csak ennyi.